src/v2/view-builder/views/captcha/CaptchaHelper.js

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) 2017, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { _ } from 'okta';
+import Enums from 'util/Enums';
+
+/**
+ *  Bind the CAPTCHA to the specified form's submit button(s). This will hijack the submit button's normal
+ *  event handler and render the CAPTCHA challenge instead. Upon solving it, the specified callback will be
+ *  invoked.
+ * @param {object} captchaConfig: The CAPTCHA configuration that includes the type and siteKey
+ * @param {Form} form: The form that will be rendering the CAPTCHA
+ * @param {Function} onCaptchaSolvedCallback: The callback to be invoked once CAPTCHA is solved
+* */ 
+export function renderCaptcha(captchaConfig, form, onCaptchaSolvedCallback) {
+  // eslint-disable-next-line no-undef
```

Comment:
Is this disabled rule still needed?

Reply:
Actually you're correct, this isn't needed anymore so I'll remove it :)